README.md

```diff
@@ -399,6 +389,22 @@ The Package name and version are output via an **echo** in the stage.
   Required. An instance of the [Package](src/com/claimvantage/sjsl/Package.groovy) bean object
   whose properties identify the package version to install.
 
+<a name="packageAliases"></a>
+#### Package Aliases
+
+See [Salesforce Project Configuration File for Packages](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev2gp_config_file.htm) for details.
+Use the example below to configure package aliases; the benefit is that it's not hard-coded on Jenkinsfile.
```

Comment:
Maybe change to "the benefit is that the package version Id (that typically changes over time) is kept out of the Jenkinsfile that should not need to change over time."

Reply:
Thanks, I've changed that.